website/client/src/components/settings/site.vue

```diff
@@ -291,14 +291,22 @@
             </li>
           </ul>
           <hr>
-          <div v-if="!user.auth.local.email">
-            <p>{{ $t('addLocalAuth') }}</p>
+          <div v-if="!user.auth.local.has_password">
```

Comment:
Is this supposed to be `hashed_password`? I'm not seeing a `has_password` field or method anywhere.